cmd/container_stop.go

```diff
@@ -23,6 +26,18 @@ var containerStopCmd = &cobra.Command{
 			panic("unreachable") // staticcheck false positive: https://staticcheck.io/docs/checks#SA5011
 		}
 
+		if containerName == "" && containerNames == "" {
+			utils.PrintlnError(fmt.Errorf("use neither --container \"<container name>\" nor --containers \"<container1 name>, <container2 name>\""))
```

Comment:
_typo_: wording => `use either --container \"<container name>\" or --containers \"<container1 name>, <container2 name>\" but not both at the same time`